From 5d6764d19d02e16d5d5a55a32b22360100dfa73b Mon Sep 17 00:00:00 2001
From: GianW <gian.cb@hotmail.com>
Date: 22/03/2014 12:14:27

First

diff --git a/listaEncadeada/src/listaencadeada/ListaEncadeada.java b/listaEncadeada/src/listaencadeada/ListaEncadeada.java
new file mode 100644
index 0000000..9683abb
--- /dev/null
+++ b/listaEncadeada/src/listaencadeada/ListaEncadeada.java
@@ -0,0 +1,15 @@
+package listaencadeada;
+
+
+/**
+ *
+ * @author Gian
+ */
+public class ListaEncadeada {
+
+    public static void main(String[] args) {
+        // TODO code application logic here       
+      new TelaInicial().setVisible(true);
+        
+    }
+}
